app/database/migrations/2015_01_16_084806_AlterTableComponentsDropTagsColumn.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
Yeh, we shouldn't worry about them till we actually release 1.0. That way our migrations aren't a mess before we've even released.